credentials/src/main/kotlin/SSI.kt

```diff
@@ -388,4 +468,14 @@ private fun issuerPublicJWK(vcJWT: String, resolver: DIDResolver): JWK {
 
   val publicKeyJWK = JWK.parse(publicKeyJwkMap)
   return publicKeyJWK
+}
+
+private fun evaluateJsonPath(vcJwt: VcJwt, path: String): String? {
+  val vc: VerifiableCredentialType =
```

Comment:
Thi JWT conversion could possibly be done outside the loop that is calling this function.

Reply:
true, this would make it faster in most cases